<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderPassUi</class>
 <widget class="QWidget" name="RenderPassUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="ShaderTextTabs">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="VertexShaderTab">
        <attribute name="title">
         <string>Vertex</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="vertexCodeEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="FragmentShaderTab">
        <attribute name="title">
         <string>Fragment</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="fragmentCodeEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
